album/nintendo-switch-download-soft-compilation-album.yaml

```diff
@@ -0,0 +1,222 @@
+Album: Nintendo Switch DOWNLOAD SOFT COMPILATION ALBUM
+Date: April 21, 2020
+#Date Added: February 5, 2024
+#URLs:
+#- 
+Color: '#ffffff'
+Cover Artists:
+- Nintendo Switch DOWNLOAD SOFT COMPILATION ALBUM
+Cover Art File Extension: png
+Groups:
+- Toby Fox
+- Bonus CDs
+- UNDERTALE
+#Banner Artists:
+#- Toby Fox
+#- Niklink (edits for wiki)
+#Banner Dimensions: 1100x191
+#Banner File Extension: png
+#Banner Style: |-
+#    image-rendering: pixelated;
+#    image-rendering: crisp-edges;
+#Wallpaper Artists:
+#- Toby Fox
+#Wallpaper File Extension: png
+#Additional Files:
+#- Title: Bandcamp Banner
+#  Files:
+#  - banner.png
+---
+Section: Little Town Hero
```

Comment:
these are on hsmusic.wiki now, this is a weird situation but to prepare for eventual interwiki support should maybe have the primary release be an entry in references beyond undertale? doesnt really matter though